requirements.txt

```diff
@@ -17,7 +17,8 @@ langdetect >=1.0.0,<=1.0.9
 dataclasses_json >=0.5.4
 autopep8 >=1.5.7
 dill ==0.3.4
-sktime >=0.5.0,<=0.8.0
+sktime >=0.11.0
+statsforecast >=0.5.3
 torch_optimizer ==0.1.0
 pmdarima >=1.8.0,<=1.8.3
```

Comment:
Also, we should remove this dependency.

Reply:
Right, fixed in [9787398](https://github.com/mindsdb/lightwood/pull/861/commits/9787398b5aaace26b50c12f5dbbc3e8b7fc914f2). :)